src/plugins/kernel_install/utils.sh

```diff
@@ -197,7 +197,7 @@ function collect_deploy_info()
   bootloader="[bootloader]=$bootloader"
 
   # Get distro
-  distro=$(cat /etc/*-release | grep -w 'ID\(_LIKE\)\?' | cut -d = -f 2 | xargs echo)
+  distro=$(grep < /etc/*-release --word-regexp 'ID\(_LIKE\)\?' | cut --delimiter = --fields 2 | xargs printf)
```

Comment:
The use of `xargs printf` results in the following error:

```
printf: missing operand
```

If I am not mistaken, the use of `xargs` is unnecessary, but please confirm if this is the case.

Reply:
I notice that without xargs, the output of `printf '%s' "$bootloader $distro"` in the line 204 (where the `distro` variable is used) will have a break line in the middle of string that is not "human readable". So I decided to continue using the xargs but fixing the printf error and make it been must similar as possible as echo, but I didn't find a solution for add the break-line in the final unless adding a `&& printf '\n'` in the final of command of line 200. So I`d like to ask if addind this extra `printf` will violate some coding style error or not.